core/deployment/src/main/java/io/quarkiverse/langchain4j/deployment/AiServicesProcessor.java

```diff
@@ -359,6 +362,20 @@ public void findDeclarativeServices(CombinedIndexBuildItem indexBuildItem,
         }
     }
 
+    private DotName dotName(AnnotationValue instanceAnnotation, DotName supplierDotName, Consumer<DotName> validator) {
```

Comment:
Can we please have a better name for this method?

Reply:
changed it to `getSupplierDotName`, maybe you have an even better idea?

Reply:
Sounds good!